### DISCUSSION TITLE
Pyccel AST <-> LFortran ASR conversion

### DISCUSSION THREAD

**Discussion by ratnania** [pyccel/pyccel#2011] [Ideas]
Allow a conversion between Pyccel AST and LFortran ASR.
The latter is described as BNF grammar in
https://gitlab.com/lfortran/lfortran/blob/master/grammar/ASR.asdl

**Comment by saidctb** [comment]
according to this issue https://gitlab.com/lfortran/lfortran/-/issues/121, a lot of stuff are not supported yet

**Comment by ratnania** [comment]
Yes I agree.
Following my discussion with Ondrej, most of these devs will be available in the next months.
Having a machinery for conversion will allow LFortran developpers to contribute both sides. 
This issue is more on having a Proof Of Concept and see of we will be able to ensure this conversion.

**Comment by EmilyBourne** [comment]
Is this PR still relevant?

If so can you update the link which is now dead and add more explanation of what is wanted here. I am not familiar with LFortran ASR, or the abbreviation ASR. Is the aim to create equivalences between 2 AST representations or to print an additional language?
